airflow/example_dags/example_branch_operator.py

```diff
@@ -35,6 +35,7 @@
     dag_id='example_branch_operator',
     default_args=args,
     schedule_interval="@daily",
+    tags=['examples']
```

Comment:
```suggestion
    tags=['example']
```
What do you think about using singular form?